go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/go-git/go-git/v5 v5.11.0
 	github.com/in-toto/attestation v1.0.1
 	github.com/jonboulle/clockwork v0.4.0
-	github.com/secure-systems-lab/go-securesystemslib v0.8.0
+	github.com/secure-systems-lab/go-securesystemslib v0.8.1-0.20240108171218-da429971be5a
```

Comment:
Also, this has been merged into main. dependabot will auto-update on the next release, v0.9.0.